Description: Put all orphan files into the airportipinspector package
 Some files were not declared as being part of a package.
Author: Julien BLACHE <jblache@debian.org>

Index: airport-utils-2/ipinspector/AboutDialog.java
===================================================================
--- airport-utils-2.orig/ipinspector/AboutDialog.java	2003-01-29 22:54:15.000000000 +0100
+++ airport-utils-2/ipinspector/AboutDialog.java	2010-03-11 20:56:30.535753721 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airportipinspector;
 
 import java.awt.*;
 import java.awt.event.*;
Index: airport-utils-2/ipinspector/AirportInfoChangeEmailer.java
===================================================================
--- airport-utils-2.orig/ipinspector/AirportInfoChangeEmailer.java	2002-10-30 03:46:12.000000000 +0100
+++ airport-utils-2/ipinspector/AirportInfoChangeEmailer.java	2010-03-11 20:56:30.565172939 +0100
@@ -18,7 +18,8 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *
  */
- 
+
+package airportipinspector; 
  
 import java.io.*;
 import java.util.*;
Index: airport-utils-2/ipinspector/AirportInfoChangeListener.java
===================================================================
--- airport-utils-2.orig/ipinspector/AirportInfoChangeListener.java	2002-10-30 04:04:51.000000000 +0100
+++ airport-utils-2/ipinspector/AirportInfoChangeListener.java	2010-03-11 20:56:30.590256767 +0100
@@ -18,7 +18,8 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *
  */
- 
+
+package airportipinspector; 
 
 import airport.*;
 
Index: airport-utils-2/ipinspector/IPInspector.java
===================================================================
--- airport-utils-2.orig/ipinspector/IPInspector.java	2003-01-30 21:21:44.000000000 +0100
+++ airport-utils-2/ipinspector/IPInspector.java	2010-03-11 20:56:30.615342686 +0100
@@ -19,7 +19,7 @@
  *
  */
  
-
+package airportipinspector;
 
 
 import java.util.*;
Index: airport-utils-2/ipinspector/JTextAreaWriter.java
===================================================================
--- airport-utils-2.orig/ipinspector/JTextAreaWriter.java	2002-10-30 03:45:50.000000000 +0100
+++ airport-utils-2/ipinspector/JTextAreaWriter.java	2010-03-11 20:56:30.640431898 +0100
@@ -18,7 +18,8 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *
  */
- 
+
+package airportipinspector; 
  
 import java.io.*;
 import javax.swing.text.*;
Index: airport-utils-2/ipinspector/MailerThread.java
===================================================================
--- airport-utils-2.orig/ipinspector/MailerThread.java	2003-01-30 21:30:33.000000000 +0100
+++ airport-utils-2/ipinspector/MailerThread.java	2010-03-11 20:56:30.665752284 +0100
@@ -18,7 +18,8 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *
  */
- 
+
+package airportipinspector; 
 
 import java.io.*;
 import java.util.*;
Index: airport-utils-2/ipinspector/Message.java
===================================================================
--- airport-utils-2.orig/ipinspector/Message.java	2002-10-30 03:44:53.000000000 +0100
+++ airport-utils-2/ipinspector/Message.java	2010-03-11 20:56:30.690752261 +0100
@@ -19,6 +19,7 @@
  *
  */
  
+package airportipinspector;
  
  /* Message class
 
Index: airport-utils-2/ipinspector/POPWeasel.java
===================================================================
--- airport-utils-2.orig/ipinspector/POPWeasel.java	2002-10-30 03:44:17.000000000 +0100
+++ airport-utils-2/ipinspector/POPWeasel.java	2010-03-11 20:56:30.715752110 +0100
@@ -18,6 +18,8 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *
  */
+
+package airportipinspector;
  
  /* class POPWeasel
 
Index: airport-utils-2/ipinspector/Preferences.java
===================================================================
--- airport-utils-2.orig/ipinspector/Preferences.java	2002-10-30 03:44:40.000000000 +0100
+++ airport-utils-2/ipinspector/Preferences.java	2010-03-11 20:56:30.757437080 +0100
@@ -18,6 +18,8 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *
  */
+
+package airportipinspector;
  
  /* Preferences class
 
Index: airport-utils-2/ipinspector/PreferencesDialog.java
===================================================================
--- airport-utils-2.orig/ipinspector/PreferencesDialog.java	2003-01-29 22:47:56.000000000 +0100
+++ airport-utils-2/ipinspector/PreferencesDialog.java	2010-03-11 20:56:30.782518180 +0100
@@ -19,7 +19,7 @@
  *
  */
  
- 
+package airportipinspector; 
 
 import java.awt.*;
 import javax.swing.*;
